crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -2206,21 +2315,21 @@ type SplitCoinsTransaction {
 
 enum StakeStatus {
 	"""
-	The stake object is active in a staking pool and it is generating rewards
+	The stake object is active in a staking pool and it is generating rewards.
 	"""
 	ACTIVE
 	"""
-	The stake awaits to join a staking pool in the next epoch
+	The stake awaits to join a staking pool in the next epoch.
 	"""
 	PENDING
 	"""
-	The stake is no longer active in any staking pool
+	The stake is no longer active in any staking pool.
 	"""
 	UNSTAKED
 }
 
 """
-Parameters related to subsiding staking rewards
+Parameters related to subsiding staking rewards.
```

Comment:
here's another instance where i feel like one can just look at the comments for its fields to understand what StakeSubsidy is